website/themes/diflabs/assets/css/main.css

```diff
@@ -214,6 +212,10 @@ a:hover {
     border-bottom: 2px dashed #98FB98; /* Change underline color on hover */
 }
 
+a:hover {
+    color: #66FF66;; /* Change hover color to yellow */
```

Comment:
This is a different shade of green, not yellow as the comment suggests

Reply:
yes, this is correct.